client/package.json

```diff
@@ -8,6 +8,7 @@
     "@testing-library/user-event": "^13.5.0",
```

Comment:
![IMG_C8FAEE7FA76D-1](https://github.com/ylopez25/Tech-dive/assets/79764816/984fbbfc-db64-45e7-8e02-9795bcb723a0)
Looked at your dependencies, I plan on using chakraUi so the <link> tag for me is now <ChakraLink>, I am unsure if you are still using it

Reply:
need to look into this more before incorporating it . Will add it for exam list issue